src/src/pages/capabilities/MyCapabilities.js

```diff
@@ -68,6 +70,89 @@ export default function MyCapabilities() {
   useEffect(() => {
     setShowAwsResourcesSpinner(isLoadingAwsResourcesCounts);
   }, [isLoadingAwsResourcesCounts]);
+
+
+  const columns = useMemo(
+    () => [
+      {
+        accessorFn: (row) => { return { name: row.name, description: row.description } },
+        header: 'Name',
+        size: 350,
+        enableColumnFilterModes: false,
+        Cell: ({ cell }) => {
+          return <div> <Text styledAs="action" as={"div"}>
+            {cell.getValue().name}
+          </Text>
+            <Text styledAs="caption" as={"div"}>
+              {cell.getValue().description}
+            </Text></div>
+        }
+
+      },
+      {
+        accessorKey: 'awsResources',
+        header: 'Aws Resources',
+        size: 150,
+        enableColumnFilterModes: false,
+        muiTableHeadCellProps: {
+          align: 'center'
+        },
+        muiTableBodyCellProps: {
+          align: 'center'
+        }
+        
```

Comment:
I will suggest using the `InlineAwsCountSummary` in a Cell-definition as for the other columns.
Firstly this is already defined and does some simple rendering with a logo, secondly we offload this rendering to a specialized component just as we do for costs.

This is not necessary at this point, though